README.md

```diff
@@ -1,3 +1,125 @@
 # Creepy
 
-Creepy or *Call reemote python* is a lightweight library for remote procedure calls from trusted callers.
+Creepy, or **C**all **REE**mote **PY**thon, is a lightweight library for secure remote procedure calls (RPC) from trusted callers. It enables seamless execution of Python code across different machines or processes with a focus on security.
+
+## Features
+
+### Core Functionality
+
+- **Secure Remote Procedure Calls**: Execute Python functions on remote servers with end-to-end encryption
+- **Proxy Objects**: Transparent interaction with remote objects as if they were local
+- **Secure Communication**: Encrypted channels using ChaCha20Poly1305 and asymmetric key exchange
+- **Session Management**: Persistent connections with automatic session handling
+- **Memory Protection**: Secure storage for sensitive data with protections against memory leakage
+
+### Security Features
+
+- **SecureString**: Memory-protected string implementation for sensitive data
+- **Memory Locking**: Prevent sensitive data from being swapped to disk
+- **Secure Channels**: Encrypted communication between client and server
+- **Key Exchange**: Secure handshake protocol for establishing trusted connections
+
+### Utilities
+
+- **Process Isolation**: Run code in separate processes for improved security
+- **Module Import**: Import and use remote modules as if they were local
```

Comment:
I would say it's just a part of creepy functionality rather than an utility, so let's remove this.
```suggestion
```

Reply:
We actually have `import_module` method which does what is said, so I don't really understand what's wrong with this line.